### ISSUE TITLE
#165 reintroducing webpack issue

### ISSUE THREAD

**Issue by bopm** [stimulusreflex/cable_ready#178]
#165 is reverting changes from #169 which is leading to next error in webpack@5:
```
ERROR in ./node_modules/stimulus_reflex/node_modules/cable_ready/javascript/index.js 28:2-9
Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)
```

**Comment by bopm** [comment]
@leastbad I am going to tag you here as you were approver of #169 in the first place.

**Comment by marcoroth** [comment]
Hey @bopm, thank you for spotting this! This somehow got lost in a merge conflict. 

I cherry-picked the commit from #169 as https://github.com/stimulusreflex/cable_ready/commit/1bded053b64e8f497fcf368a258b0326ea4c886f on master!

**Comment by bopm** [comment]
@marcoroth Any chance for a new version release? As currently released npm version still lacks this change and contains that code inside of node_modules inside of stimulus_reflex.

**Comment by marcoroth** [comment]
Hey @bopm, sorry it took us a while to get this straight. We just released CableReady `5.0.0.pre9` which includes this fix. I'm sorry if this caused any inconvenience in the meantime.
